templates/terraform/aws_agent_with_opentelemetry_collector/README.md

```diff
@@ -6,12 +6,12 @@ This Terraform module deploys a Monte Carlo Agent, Data Store, and OpenTelemetry
 
 This module creates:
 - A Monte Carlo Agent using the official Terraform module (includes S3 storage bucket)
-- An OpenTelemetry Collector using the local `aws_otel_collector` module
+- An OpenTelemetry Collector using the `monte-carlo-data/otel-collector/aws` module
```

Comment:
I was expecting a different name in the module README: https://github.com/monte-carlo-data/terraform-aws-otel-collector/blob/main/README.md. That will need to be updated.